panic in `ddev describe` in  1.18.0-rc1  with mutagen enabled globally

**Run a Diagnostic and Paste Link Here**

https://gist.github.com/marcoscano/60bb920bd10674a9166a86b1254df1a9

**Describe the bug**

Just upgraded from alpha4 into rc1:

```
$ ddev poweroff
$ brew upgrade drud/ddev-edge/ddev
...
==> Upgrading 1 outdated package:
drud/ddev-edge/ddev 1.18.0-alpha4 -> 1.18.0-rc1
==> Upgrading drud/ddev-edge/ddev
  1.18.0-alpha4 -> 1.18.0-rc1 
... 
```

Then `ddev start` and `ddev restart` work fine.

However, when I try `ddev describe` I get:

```
$ ddev describe
tWidth=279 urlPortWidth=111.6 infoWidth=69 
panic: interface conversion: interface {} is nil, not string

goroutine 1 [running]:
github.com/drud/ddev/cmd/ddev/cmd.renderAppDescribe(0xc000438300, 0xc000110000, 0xc000110000, 0x0, 0x0, 0x1)
	/workdir/cmd/ddev/cmd/describe.go:143 +0x27df
github.com/drud/ddev/cmd/ddev/cmd.glob..func14(0x1ab93a0, 0x1af7680, 0x0, 0x0)
	/workdir/cmd/ddev/cmd/describe.go:52 +0x185
github.com/spf13/cobra.(*Command).execute(0x1ab93a0, 0x1af7680, 0x0, 0x0, 0x1ab93a0, 0x1af7680)
	/workdir/vendor/github.com/spf13/cobra/command.go:854 +0x2c2
github.com/spf13/cobra.(*Command).ExecuteC(0x1abcac0, 0x1f5, 0x0, 0x0)
	/workdir/vendor/github.com/spf13/cobra/command.go:958 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
	/workdir/vendor/github.com/spf13/cobra/command.go:895
github.com/drud/ddev/cmd/ddev/cmd.Execute()
	/workdir/cmd/ddev/cmd/root.go:151 +0x3c
main.main()
	/workdir/cmd/ddev/main.go:21 +0xd9
```

**To Reproduce**
<!--
Steps to reproduce the behavior:
-->

**Expected behavior**
<!--
A clear and concise description of what you expected to happen.
-->

**Screenshots**
<!--
If applicable, add screenshots to help explain your problem.
-->

**Version and configuration information (please complete the following information):**

- Host computer OS and Version: macOS BigSur 11.5.2
- Docker Desktop version if on macOS or Windows: 3.6.0
- ddev version information (use `ddev version`)

```
$ ddev version
 ITEM            VALUE                                                            
 DDEV version    v1.18.0-rc1                                                      
 architecture    amd64                                                            
 db              drud/ddev-dbserver-mariadb-10.3:20210729_cspitzlay_mysql_history 
 dba             phpmyadmin:5                                                     
 ddev-ssh-agent  drud/ddev-ssh-agent:v1.17.0                                      
 docker          20.10.8                                                          
 docker-compose  1.29.2                                                           
 mutagen         0.12.0-beta5                                                     
 os              darwin                                                           
 router          drud/ddev-router:v1.17.6                                         
 web             drud/ddev-webserver:20210823_bump_upstream_webserver 
```

- config.yaml contents for the misbehaving project (the part before the comments)

```
name: foobar
type: drupal9
docroot: web
php_version: "7.4"
webserver_type: nginx-fpm
router_http_port: "9080"
router_https_port: "9443"
xdebug_enabled: false
additional_hostnames: []
additional_fqdns: []
mariadb_version: "10.4"
mysql_version: ""
nfs_mount_enabled: false
mutagen_enabled: false
use_dns_when_possible: true
disable_settings_management: true
timezone: "America/Los_Angeles"
composer_version: "2"
web_environment: []
```

- Do you have any custom configuration (nginx, php, mysql) in the .ddev folder? If so, have you tried without them?

The only thing I can see is php with 
```
[PHP]
memory_limit = -1;
```

**Additional context**

I have mutagen enabled globally, not on a per-project basis.

--- a/cmd/ddev/cmd/config-global.go
+++ b/cmd/ddev/cmd/config-global.go
@@ -2,6 +2,7 @@ package cmd
 
 import (
 	"github.com/drud/ddev/pkg/globalconfig"
+	"github.com/drud/ddev/pkg/nodeps"
 	"github.com/drud/ddev/pkg/output"
 	"github.com/drud/ddev/pkg/util"
 	"github.com/drud/ddev/pkg/version"
@@ -104,8 +105,12 @@ func handleGlobalConfig(cmd *cobra.Command, args []string) {
 
 	if cmd.Flag("table-style").Changed {
 		val, _ := cmd.Flags().GetString("table-style")
-		globalconfig.DdevGlobalConfig.TableStyle = val
-		dirty = true
+		if nodeps.ArrayContainsString(globalconfig.ValidTableStyleList(), val) {
+			globalconfig.DdevGlobalConfig.TableStyle = val
+			dirty = true
+		} else {
+			util.Error("table-style=%s is not valid. Valid options include %s. Not changing table-style.\n", val, strings.Join(globalconfig.ValidTableStyleList(), ", "))
+		}
 	}
 
 	if cmd.Flag("auto-restart-containers").Changed {

--- a/cmd/ddev/cmd/describe.go
+++ b/cmd/ddev/cmd/describe.go
@@ -71,7 +71,7 @@ func renderAppDescribe(app *ddevapp.DdevApp, desc map[string]interface{}) (strin
 		urlPortWidth = float64(tWidth) / urlPortWidthFactor
 		infoWidth = tWidth / 4
 	}
-	output.UserOut.Debugf("tWidth=%v urlPortWidth=%v infoWidth=%v", tWidth, urlPortWidth, infoWidth)
+	util.Debug("detected terminal width=%v urlPortWidth=%v infoWidth=%v", tWidth, urlPortWidth, infoWidth)
 	t.SetAllowedRowLength(tWidth)
 	t.SetColumnConfigs([]table.ColumnConfig{
 		{

--- a/cmd/ddev/cmd/describe.go
+++ b/cmd/ddev/cmd/describe.go
@@ -72,21 +72,23 @@ func renderAppDescribe(app *ddevapp.DdevApp, desc map[string]interface{}) (strin
 		infoWidth = tWidth / 4
 	}
 	util.Debug("detected terminal width=%v urlPortWidth=%v infoWidth=%v", tWidth, urlPortWidth, infoWidth)
-	t.SetAllowedRowLength(tWidth)
-	t.SetColumnConfigs([]table.ColumnConfig{
-		{
-			Name:     "Service",
-			WidthMax: 8,
-		},
-		{
-			Name:     "URL/Port",
-			WidthMax: int(urlPortWidth),
-		},
-		{
-			Name:     "Info",
-			WidthMax: infoWidth,
-		},
-	})
+	if !globalconfig.DdevGlobalConfig.SimpleFormatting {
+		t.SetAllowedRowLength(tWidth)
+		t.SetColumnConfigs([]table.ColumnConfig{
+			{
+				Name:     "Service",
+				WidthMax: 8,
+			},
+			{
+				Name:     "URL/Port",
+				WidthMax: int(urlPortWidth),
+			},
+			{
+				Name:     "Info",
+				WidthMax: infoWidth,
+			},
+		})
+	}
 	t.AppendHeader(table.Row{"Service", "Stat", "URL/Port", "Info"})
 	t.AppendRow(table.Row{"Project", "", app.Name, desc["shortroot"].(string)})
 

--- a/cmd/ddev/cmd/import-db_test.go
+++ b/cmd/ddev/cmd/import-db_test.go
@@ -35,7 +35,7 @@ func TestCmdImportDB(t *testing.T) {
 	err = app.Start()
 	require.NoError(t, err)
 
-	if app.MutagenEnabled || app.MutagenEnabledGlobal {
+	if app.IsMutagenEnabled() {
 		_, _, longStatus, _ := app.MutagenStatus()
 		t.Logf("mutagen status before show tables=%s", longStatus)
 	}
@@ -48,7 +48,7 @@ func TestCmdImportDB(t *testing.T) {
 	assert.NoError(err, "mysql exec output=%s", out)
 	require.Equal(t, "", out)
 
-	if app.MutagenEnabled || app.MutagenEnabledGlobal {
+	if app.IsMutagenEnabled() {
 		_, _, longStatus, _ := app.MutagenStatus()
 		t.Logf("mutagen status before opening file=%s", longStatus)
 	}
@@ -60,7 +60,7 @@ func TestCmdImportDB(t *testing.T) {
 	// nolint: errcheck
 	defer f.Close()
 
-	if app.MutagenEnabled || app.MutagenEnabledGlobal {
+	if app.IsMutagenEnabled() {
 		_, _, longStatus, _ := app.MutagenStatus()
 		t.Logf("mutagen status before import-db=%s", longStatus)
 	}

--- a/cmd/ddev/cmd/list_test.go
+++ b/cmd/ddev/cmd/list_test.go
@@ -151,12 +151,6 @@ func TestCmdListContinuous(t *testing.T) {
 
 	assert := asrt.New(t)
 
-	globalconfig.DdevGlobalConfig.SimpleFormatting = true
-	_ = globalconfig.WriteGlobalConfig(globalconfig.DdevGlobalConfig)
-	t.Cleanup(func() {
-		globalconfig.DdevGlobalConfig.SimpleFormatting = false
-		_ = globalconfig.WriteGlobalConfig(globalconfig.DdevGlobalConfig)
-	})
 	// Execute "ddev list --continuous"
 	cmd := oexec.Command(DdevBin, "list", "-j", "--continuous")
 	stdout, err := cmd.StdoutPipe()
@@ -167,18 +161,15 @@ func TestCmdListContinuous(t *testing.T) {
 
 	reader := bufio.NewReader(stdout)
 
-	blob := make([]byte, 16000)
-	byteCount, err := reader.Read(blob)
+	blob, err := reader.ReadBytes('\n')
 	assert.NoError(err)
+	byteCount := len(blob)
 	blob = blob[:byteCount-1]
-	require.True(t, byteCount > 300, "byteCount should have been >300 and was %v", byteCount)
+	require.True(t, byteCount > 300, "byteCount should have been >300 and was %v blob=%s", byteCount, string(blob))
 
 	f, err := unmarshalJSONLogs(string(blob))
-	if err != nil {
-		assert.NoError(err, "could not unmarshal ddev output: %v", err)
-		t.Logf("============== ddev list -j --continuous failed logs =================\n%s\n", string(blob))
-		t.FailNow()
-	}
+	require.NoError(t, err, "Could not unmarshall blob, err=%v, content=%s", err, blob)
+
 	assert.NotEmpty(f[0]["raw"])
 	time1 := f[0]["time"]
 	if len(f) > 1 {
@@ -188,13 +179,13 @@ func TestCmdListContinuous(t *testing.T) {
 	time.Sleep(time.Millisecond * 1500)
 
 	// Now read more from the pipe after resetting blob
-	blob = make([]byte, 16000)
-	byteCount, err = reader.Read(blob)
+	blob, err = reader.ReadBytes('\n')
+	byteCount = len(blob)
 	assert.NoError(err)
 	blob = blob[:byteCount-1]
-	require.True(t, byteCount > 300)
+	require.True(t, byteCount > 300, "byteCount should have been >300 and was %v blob=%s", byteCount, string(blob))
 	f, err = unmarshalJSONLogs(string(blob))
-	require.NoError(t, err)
+	require.NoError(t, err, "Could not unmarshall blob, err=%v, content=%s", err, blob)
 	if len(f) > 1 {
 		t.Logf("Expected just one line in second read, but got %d lines instead", len(f))
 	}
@@ -206,5 +197,4 @@ func TestCmdListContinuous(t *testing.T) {
 	// Kill the process we started.
 	err = cmd.Process.Kill()
 	assert.NoError(err)
-
 }

--- a/cmd/ddev/cmd/list_test.go
+++ b/cmd/ddev/cmd/list_test.go
@@ -151,6 +151,11 @@ func TestCmdListContinuous(t *testing.T) {
 
 	assert := asrt.New(t)
 
+	oldDdevDebug := os.Getenv("DDEV_DEBUG")
+	_ = os.Setenv("DDEV_DEBUG", "")
+	t.Cleanup(func() {
+		_ = os.Setenv("DDEV_DEBUG", oldDdevDebug)
+	})
 	// Execute "ddev list --continuous"
 	cmd := oexec.Command(DdevBin, "list", "-j", "--continuous")
 	stdout, err := cmd.StdoutPipe()

--- a/cmd/ddev/cmd/list_test.go
+++ b/cmd/ddev/cmd/list_test.go
@@ -25,9 +25,14 @@ func TestCmdList(t *testing.T) {
 	site := TestSites[0]
 	err := os.Chdir(site.Dir)
 
+	globalconfig.DdevGlobalConfig.SimpleFormatting = true
+	_ = globalconfig.WriteGlobalConfig(globalconfig.DdevGlobalConfig)
+
 	t.Cleanup(func() {
 		err = os.Chdir(origDir)
 		assert.NoError(err)
+		globalconfig.DdevGlobalConfig.SimpleFormatting = false
+		_ = globalconfig.WriteGlobalConfig(globalconfig.DdevGlobalConfig)
 	})
 	// This gratuitous ddev start -a repopulates the ~/.ddev/global_config.yaml
 	// project list, which has been damaged by other tests which use
@@ -146,6 +151,12 @@ func TestCmdListContinuous(t *testing.T) {
 
 	assert := asrt.New(t)
 
+	globalconfig.DdevGlobalConfig.SimpleFormatting = true
+	_ = globalconfig.WriteGlobalConfig(globalconfig.DdevGlobalConfig)
+	t.Cleanup(func() {
+		globalconfig.DdevGlobalConfig.SimpleFormatting = false
+		_ = globalconfig.WriteGlobalConfig(globalconfig.DdevGlobalConfig)
+	})
 	// Execute "ddev list --continuous"
 	cmd := oexec.Command(DdevBin, "list", "-j", "--continuous")
 	stdout, err := cmd.StdoutPipe()

--- a/cmd/ddev/cmd/mutagen-cmd_test.go
+++ b/cmd/ddev/cmd/mutagen-cmd_test.go
@@ -44,13 +44,13 @@ func TestCmdMutagen(t *testing.T) {
 		assert.NoError(err)
 		err = app.Start()
 		assert.NoError(err)
-		assert.False(app.MutagenEnabled)
+		assert.False(app.IsMutagenEnabled())
 		err = os.Chdir(origDir)
 		assert.NoError(err)
 	})
 
 	require.False(t, globalconfig.DdevGlobalConfig.MutagenEnabledGlobal)
-	require.False(t, app.MutagenEnabled)
+	require.False(t, app.IsMutagenEnabled())
 
 	_, err = exec.RunHostCommand(DdevBin, "config", "--mutagen-enabled=true")
 	assert.NoError(err)
@@ -60,7 +60,7 @@ func TestCmdMutagen(t *testing.T) {
 	require.NoError(t, err)
 
 	// Make sure it got turned on
-	assert.True(app.MutagenEnabled)
+	assert.True(app.IsMutagenEnabled())
 
 	// Now test subcommands. Wait just a bit for mutagen to get completely done, with transition problems sorted outx
 	err = app.StartAndWait(10)
@@ -94,7 +94,7 @@ func TestCmdMutagen(t *testing.T) {
 	require.NoError(t, err)
 
 	// Make sure it got turned off
-	assert.False(app.MutagenEnabled)
+	assert.False(app.IsMutagenEnabled())
 
 	_, err = exec.RunHostCommand(DdevBin, "config", "global", "--mutagen-enabled=true")
 	assert.NoError(err)

--- a/cmd/ddev/cmd/mutagen-monitor.go
+++ b/cmd/ddev/cmd/mutagen-monitor.go
@@ -24,7 +24,7 @@ var MutagenMonitorCmd = &cobra.Command{
 		if err != nil {
 			util.Failed("Failed to get active project: %v", err)
 		}
-		if !(app.MutagenEnabled || app.MutagenEnabledGlobal) {
+		if !(app.IsMutagenEnabled()) {
 			util.Failed("Mutagen is not enabled on project %s", app.Name)
 		}
 		ddevapp.MutagenMonitor(app)

--- a/cmd/ddev/cmd/mutagen-reset.go
+++ b/cmd/ddev/cmd/mutagen-reset.go
@@ -27,7 +27,7 @@ var MutagenResetCmd = &cobra.Command{
 		if err != nil {
 			util.Failed("Failed to get active project: %v", err)
 		}
-		if !(app.MutagenEnabled || app.MutagenEnabledGlobal) {
+		if !(app.IsMutagenEnabled()) {
 			util.Failed("Mutagen is not enabled on project %s", app.Name)
 		}
 

--- a/cmd/ddev/cmd/mutagen-status.go
+++ b/cmd/ddev/cmd/mutagen-status.go
@@ -30,7 +30,7 @@ var MutagenStatusCmd = &cobra.Command{
 		if err != nil {
 			util.Failed("Failed to get active project: %v", err)
 		}
-		if !(app.MutagenEnabled || app.MutagenEnabledGlobal) {
+		if !(app.IsMutagenEnabled()) {
 			util.Failed("Mutagen is not enabled on project %s", app.Name)
 		}
 		status, shortResult, longResult, _ := app.MutagenStatus()

--- a/cmd/ddev/cmd/mutagen-sync.go
+++ b/cmd/ddev/cmd/mutagen-sync.go
@@ -30,7 +30,7 @@ var MutagenSyncCmd = &cobra.Command{
 		if err != nil {
 			util.Failed("Failed to get active project: %v", err)
 		}
-		if !(app.MutagenEnabled || app.MutagenEnabledGlobal) {
+		if !(app.IsMutagenEnabled()) {
 			util.Failed("Mutagen is not enabled on project %s", app.Name)
 		}
 

--- a/docs/users/cli-usage.md
+++ b/docs/users/cli-usage.md
@@ -8,8 +8,8 @@ Each of these commands has full help. For example, `ddev start -h` or `ddev help
 
 * `ddev config` configures a project for ddev, creating a .ddev directory according to your responses. It should be executed in the project (repository) root.
 * `ddev start` and `ddev stop` start and stop the containers that comprise a project. `ddev restart` just does a stop and a start. `ddev poweroff` stops all ddev-related containers and projects.
-* `ddev describe` or `ddev describe <projectname>` gives you full details about the project, what ports it uses, how to access them, etc.
-* `ddev list` shows running projects.
+* `ddev describe` or `ddev describe <projectname>` gives you full details about the project, what ports it uses, how to access them, etc. (Change the format with `ddev config global --table-style=bright` or `bold` or `default`)
+* `ddev list` shows running projects. (Change the format with `ddev config global --table-style=bright` or `bold` or `default`)
 * `ddev mysql` gives direct access to the mysql client
 * `ddev sequelpro`, `ddev sequelace`, and `ddev tableplus` (macOS only, if the app is installed) give access to the Sequel Pro, Sequel Ace, or TablePlus database browser GUIs.
 * `ddev heidisql` (Windows/WSL2 only, if installed) gives access to the HeidiSQL database browser GUI.
@@ -25,6 +25,7 @@ Each of these commands has full help. For example, `ddev start -h` or `ddev help
 * `ddev drush` (Drupal and Backdrop only) gives direct access to the drush CLI
 * `ddev artisan` (Laravel only) gives direct access to the Laravel artisan CLI
 * `ddev magento` (Magento2 only) gives access to the magento CLI
+* `ddev config global --simple-formatting` tells ddev to not try to make a fancy table in `ddev describe` and `ddev list`
 
 ## Partial Bundled Tools List
 

--- a/docs/users/extend/config_yaml.md
+++ b/docs/users/extend/config_yaml.md
@@ -55,6 +55,9 @@ The $HOME/.ddev/global_config.yaml has a few key global config options.
 | Item  | Description   | Notes  |
 |---|---|---|
 | nfs_mount_enabled | Enables NFS mounting globally for all projects | Only a "true" value has any effect. If true, NFS will be used on all projects, regardless of any settings in the individual projects. |
+| mutagen_enabled | Enables Mutagen asynchronous caching globally for all projects | If true, Mutagen will be used on all projects, and this overrides NFS mounting as it's incompatible with NFS. |
+| table-style | Set the style for `ddev list` and `ddev describe`| Options are "default", "bold", and "bright". "bright" is a pleasant colored output that some people will like. |
+| simple-formatting | If true, turns off most table formatting in `ddev list` and `ddev describe` | This tries to give plain vanilla output for list and describe, and also suppresses colorized text everywhere. |
 | fail_on_hook_fail | Enables `ddev start` interruption globally for all projects when a hook fails | Decide whether `ddev start` should be interrupted by a failing hook |
 | omit_containers | Allows the project to not load specified containers | For example, `omit_containers: [ "dba", "ddev-ssh-agent"]`. Currently only these containers are supported. Note that you cannot omit the "db" container in the global configuration, but you can in the per-project .ddev/config.yaml. |
 | web_environment | Inject environment variables into web container | For example, `web_environment: ["SOMEENV=someval", "SOMEOTHERENV=someotherval"]`.  |

--- a/pkg/ddevapp/config.go
+++ b/pkg/ddevapp/config.go
@@ -722,9 +722,9 @@ func (app *DdevApp) RenderComposeYAML() (string, error) {
 		OmitRouter:                nodeps.ArrayContainsString(app.GetOmittedContainers(), globalconfig.DdevRouterContainer),
 		OmitSSHAgent:              nodeps.ArrayContainsString(app.GetOmittedContainers(), "ddev-ssh-agent"),
 		BindAllInterfaces:         app.BindAllInterfaces,
-		MutagenEnabled:            (app.MutagenEnabled || app.MutagenEnabledGlobal),
+		MutagenEnabled:            (app.IsMutagenEnabled()),
 
-		NFSMountEnabled:       (app.NFSMountEnabled || app.NFSMountEnabledGlobal) && !app.MutagenEnabled,
+		NFSMountEnabled:       (app.NFSMountEnabled || app.NFSMountEnabledGlobal) && !app.IsMutagenEnabled(),
 		NFSSource:             "",
 		IsWindowsFS:           runtime.GOOS == "windows",
 		NoProjectMount:        app.NoProjectMount,
@@ -764,7 +764,7 @@ func (app *DdevApp) RenderComposeYAML() (string, error) {
 		}
 	}
 
-	if app.MutagenEnabled || app.MutagenEnabledGlobal {
+	if app.IsMutagenEnabled() {
 		templateVars.MountType = "volume"
 		templateVars.WebMount = "project_mutagen"
 		templateVars.MutagenVolumeName = GetMutagenVolumeName(app)

--- a/pkg/ddevapp/ddevapp.go
+++ b/pkg/ddevapp/ddevapp.go
@@ -257,8 +257,8 @@ func (app *DdevApp) Describe(short bool) (map[string]interface{}, error) {
 	appDesc["router_disabled"] = IsRouterDisabled(app)
 	appDesc["primary_url"] = app.GetPrimaryURL()
 	appDesc["type"] = app.GetType()
-	appDesc["mutagen_enabled"] = app.MutagenEnabled || app.MutagenEnabledGlobal
-	if app.MutagenEnabled {
+	appDesc["mutagen_enabled"] = app.IsMutagenEnabled()
+	if app.IsMutagenEnabled() {
 		appDesc["mutagen_status"], _, _, err = app.MutagenStatus()
 		if err != nil {
 			appDesc["mutagen_status"] = err.Error() + " " + appDesc["mutagen_status"].(string)
@@ -271,7 +271,7 @@ func (app *DdevApp) Describe(short bool) (map[string]interface{}, error) {
 	}
 	appDesc["hostname"] = app.GetHostname()
 	appDesc["hostnames"] = app.GetHostnames()
-	appDesc["nfs_mount_enabled"] = (app.NFSMountEnabled || app.NFSMountEnabledGlobal) && !(app.MutagenEnabled || app.MutagenEnabledGlobal)
+	appDesc["nfs_mount_enabled"] = (app.NFSMountEnabled || app.NFSMountEnabledGlobal) && !(app.IsMutagenEnabled())
 	appDesc["fail_on_hook_fail"] = app.FailOnHookFail || app.FailOnHookFailGlobal
 	httpURLs, httpsURLs, allURLs := app.GetAllURLs()
 	appDesc["httpURLs"] = httpURLs
@@ -994,7 +994,7 @@ func (app *DdevApp) Start() error {
 		return err
 	}
 
-	if app.MutagenEnabled || app.MutagenEnabledGlobal {
+	if app.IsMutagenEnabled() {
 		mounted, err := IsMutagenVolumeMounted(app)
 		if err != nil {
 			return err

--- a/pkg/ddevapp/ddevapp.go
+++ b/pkg/ddevapp/ddevapp.go
@@ -158,6 +158,7 @@ func List(activeOnly bool, continuous bool, continuousSleepTime int) {
 			}
 			tWidth, _ := nodeps.GetTerminalWidthHeight()
 			t.SetAllowedRowLength(tWidth)
+			util.Debug("detected terminal width=%v", tWidth)
 			t.SortBy([]table.SortBy{{Name: "Name"}})
 			t.AppendFooter(table.Row{
 				"Router", "", routerStatus},

--- a/pkg/ddevapp/ddevapp.go
+++ b/pkg/ddevapp/ddevapp.go
@@ -6,7 +6,6 @@ import (
 	"fmt"
 	"github.com/drud/ddev/pkg/globalconfig"
 	"github.com/drud/ddev/pkg/nodeps"
-	"github.com/jedib0t/go-pretty/v6/table"
 	"github.com/lextoumbourou/goodhosts"
 	"github.com/mattn/go-isatty"
 	"github.com/pkg/errors"
@@ -119,69 +118,6 @@ type DdevApp struct {
 	DisableSettingsManagement bool                   `yaml:"disable_settings_management,omitempty"`
 	WebEnvironment            []string               `yaml:"web_environment"`
 	ComposeYaml               map[string]interface{} `yaml:"-"`
-}
-
-// List provides the functionality for `ddev list`
-// activeOnly if true only shows projects that are currently docker containers
-// continuous if true keeps requesting and outputting continuously
-// continuousSleepTime is the time between reports
-func List(activeOnly bool, continuous bool, continuousSleepTime int) {
-	runTime := util.TimeTrack(time.Now(), "ddev list")
-	defer runTime()
-
-	var out bytes.Buffer
-
-	for {
-		apps, err := GetProjects(activeOnly)
-		if err != nil {
-			util.Failed("failed getting GetProjects: %v", err)
-		}
-		appDescs := make([]map[string]interface{}, 0)
-
-		if len(apps) < 1 {
-			output.UserOut.WithField("raw", appDescs).Println("No ddev projects were found.")
-		} else {
-			t := CreateAppTable(&out)
-			for _, app := range apps {
-				desc, err := app.Describe(true)
-				if err != nil {
-					util.Error("Failed to describe project %s: %v", app.GetName(), err)
-				}
-				appDescs = append(appDescs, desc)
-				RenderAppRow(t, desc)
-			}
-
-			routerStatus, _ := GetRouterStatus()
-			var extendedRouterStatus = RenderRouterStatus()
-			if nodeps.ArrayContainsString(globalconfig.DdevGlobalConfig.OmitContainersGlobal, globalconfig.DdevRouterContainer) {
-				extendedRouterStatus = "disabled"
-			}
-			tWidth, _ := nodeps.GetTerminalWidthHeight()
-			t.SetAllowedRowLength(tWidth)
-			util.Debug("detected terminal width=%v", tWidth)
-			t.SortBy([]table.SortBy{{Name: "Name"}})
-			t.AppendFooter(table.Row{
-				"Router", "", routerStatus},
-			)
-			t.Render()
-			output.UserOut.WithField("raw", appDescs).Print(out.String())
-			if routerStatus != "healthy" {
-				rawResult := map[string]string{
-					"routerStatus":         routerStatus,
-					"extendedRouterStatus": extendedRouterStatus,
-				}
-				rawResult["routerStatus"] = routerStatus
-				rawResult["extendedStatus"] = extendedRouterStatus
-				output.UserOut.WithField("raw", rawResult)
-			}
-		}
-
-		if !continuous {
-			break
-		}
-
-		time.Sleep(time.Duration(continuousSleepTime) * time.Second)
-	}
 }
 
 // GetType returns the application type as a (lowercase) string

--- a/pkg/ddevapp/ddevapp.go
+++ b/pkg/ddevapp/ddevapp.go
@@ -1438,10 +1438,6 @@ func (app *DdevApp) DockerEnv() {
 
 	// Find out terminal dimensions
 	columns, lines := nodeps.GetTerminalWidthHeight()
-	if columns == 0 {
-		columns = 80
-		lines = 24
-	}
 
 	envVars["COLUMNS"] = strconv.Itoa(columns)
 	envVars["LINES"] = strconv.Itoa(lines)

--- a/pkg/ddevapp/ddevapp.go
+++ b/pkg/ddevapp/ddevapp.go
@@ -377,8 +377,9 @@ func (app *DdevApp) Describe(short bool) (map[string]interface{}, error) {
 			for _, e := range c.Config.Env {
 				split := strings.SplitN(e, "=", 2)
 				envName := split[0]
-				envVal := split[1]
-				if envName == "HTTP_EXPOSE" || envName == "HTTPS_EXPOSE" {
+				if len(split) == 2 && (envName == "HTTP_EXPOSE" || envName == "HTTPS_EXPOSE") {
+					envVal := split[1]
+
 					envValStr := fmt.Sprintf("%s", envVal)
 					portSpecs := strings.Split(envValStr, ",")
 					// There might be more than one exposed UI port, but this only handles the first listed,

--- a/pkg/ddevapp/ddevapp_test.go
+++ b/pkg/ddevapp/ddevapp_test.go
@@ -2,14 +2,12 @@ package ddevapp_test
 
 import (
 	"bufio"
-	"bytes"
 	"fmt"
 	"net"
 	"net/url"
 	"os"
 	"path"
 	"path/filepath"
-	"regexp"
 	"runtime"
 	"sort"
 	"strconv"
@@ -2699,82 +2697,6 @@ func TestRouterNotRunning(t *testing.T) {
 	}
 }
 
-// TestListWithoutDir prevents regression where ddev list panics if one of the
-// sites found is missing a directory
-func TestListWithoutDir(t *testing.T) {
-	// Can't run with mutagen because we actually delete the alpha
-	if runtime.GOOS == "windows" || nodeps.MutagenEnabledDefault == true {
-		t.Skip("Skipping because unreliable on Windows and can't be used with mutagen")
-	}
-	// Set up tests and give ourselves a working directory.
-	assert := asrt.New(t)
-	testcommon.ClearDockerEnv()
-	origDir, _ := os.Getwd()
-
-	// startCount is the count of apps at the start of this adventure
-	apps := ddevapp.GetActiveProjects()
-	startCount := len(apps)
-
-	testDir := testcommon.CreateTmpDir(t.Name())
-	defer testcommon.CleanupDir(testDir)
-
-	err := os.MkdirAll(testDir+"/sites/default", 0777)
-	assert.NoError(err)
-	err = os.Chdir(testDir)
-	assert.NoError(err)
-
-	app, err := ddevapp.NewApp(testDir, true)
-	assert.NoError(err)
-
-	t.Cleanup(func() {
-		err = os.Chdir(origDir)
-		assert.NoError(err)
-		err = app.Stop(true, false)
-		assert.NoError(err)
-		err = os.RemoveAll(testDir)
-		assert.NoError(err)
-	})
-	app.Name = t.Name()
-	app.Type = nodeps.AppTypeDrupal7
-	err = app.WriteConfig()
-	assert.NoError(err)
-
-	// Do a start on the configured site.
-	app, err = ddevapp.GetActiveApp("")
-	assert.NoError(err)
-	err = app.Start()
-	assert.NoError(err)
-
-	err = os.Chdir(origDir)
-	assert.NoError(err)
-
-	err = os.RemoveAll(testDir)
-	assert.NoError(err)
-
-	apps = ddevapp.GetActiveProjects()
-
-	assert.EqualValues(len(apps), startCount+1)
-
-	// Make a whole table and make sure our app directory missing shows up.
-	// This could be done otherwise, but we'd have to go find the site in the
-	// array first.
-	out := bytes.Buffer{}
-	table := ddevapp.CreateAppTable(&out)
-	for _, site := range apps {
-		desc, err := site.Describe(false)
-		if err != nil {
-			t.Fatalf("Failed to describe site %s: %v", site.GetName(), err)
-		}
-
-		ddevapp.RenderAppRow(table, desc)
-	}
-	table.Render()
-	assert.Regexp(regexp.MustCompile("(?s)"+ddevapp.SiteDirMissing+".*"+testDir), out.String())
-
-	err = app.Stop(true, false)
-	assert.NoError(err)
-}
-
 type URLRedirectExpectations struct {
 	scheme              string
 	uri                 string
@@ -3626,12 +3548,6 @@ func TestCustomCerts(t *testing.T) {
 	// If we had the regular cert, there would be several things here including *.ddev.site
 	// But e should only see the hostname listed.
 	assert.Equal(app.GetHostname(), stdout)
-}
-
-// TestDdevList tests the ddevapp.List() functionality
-// It's only here for profiling at this point.
-func TestDdevList(t *testing.T) {
-	ddevapp.List(true, false, 1)
 }
 
 // TestEnvironmentVariables tests to make sure that documented environment variables appear

--- a/pkg/ddevapp/list.go
+++ b/pkg/ddevapp/list.go
@@ -1,0 +1,74 @@
+package ddevapp
+
+import (
+	"bytes"
+	"github.com/drud/ddev/pkg/globalconfig"
+	"github.com/drud/ddev/pkg/nodeps"
+	"github.com/drud/ddev/pkg/output"
+	"github.com/drud/ddev/pkg/util"
+	"github.com/jedib0t/go-pretty/v6/table"
+	"time"
+)
+
+// List provides the functionality for `ddev list`
+// activeOnly if true only shows projects that are currently docker containers
+// continuous if true keeps requesting and outputting continuously
+// continuousSleepTime is the time between reports
+func List(activeOnly bool, continuous bool, continuousSleepTime int) {
+	runTime := util.TimeTrack(time.Now(), "ddev list")
+	defer runTime()
+
+	var out bytes.Buffer
+
+	for {
+		apps, err := GetProjects(activeOnly)
+		if err != nil {
+			util.Failed("failed getting GetProjects: %v", err)
+		}
+		appDescs := make([]map[string]interface{}, 0)
+
+		if len(apps) < 1 {
+			output.UserOut.WithField("raw", appDescs).Println("No ddev projects were found.")
+		} else {
+			t := CreateAppTable(&out)
+			for _, app := range apps {
+				desc, err := app.Describe(true)
+				if err != nil {
+					util.Error("Failed to describe project %s: %v", app.GetName(), err)
+				}
+				appDescs = append(appDescs, desc)
+				RenderAppRow(t, desc)
+			}
+
+			routerStatus, _ := GetRouterStatus()
+			var extendedRouterStatus = RenderRouterStatus()
+			if nodeps.ArrayContainsString(globalconfig.DdevGlobalConfig.OmitContainersGlobal, globalconfig.DdevRouterContainer) {
+				extendedRouterStatus = "disabled"
+			}
+			tWidth, _ := nodeps.GetTerminalWidthHeight()
+			t.SetAllowedRowLength(tWidth)
+			util.Debug("detected terminal width=%v", tWidth)
+			t.SortBy([]table.SortBy{{Name: "Name"}})
+			t.AppendFooter(table.Row{
+				"Router", "", routerStatus},
+			)
+			t.Render()
+			output.UserOut.WithField("raw", appDescs).Print(out.String())
+			if routerStatus != "healthy" {
+				rawResult := map[string]string{
+					"routerStatus":         routerStatus,
+					"extendedRouterStatus": extendedRouterStatus,
+				}
+				rawResult["routerStatus"] = routerStatus
+				rawResult["extendedStatus"] = extendedRouterStatus
+				output.UserOut.WithField("raw", rawResult)
+			}
+		}
+
+		if !continuous {
+			break
+		}
+
+		time.Sleep(time.Duration(continuousSleepTime) * time.Second)
+	}
+}

--- a/pkg/ddevapp/list.go
+++ b/pkg/ddevapp/list.go
@@ -5,6 +5,7 @@ import (
 	"github.com/drud/ddev/pkg/globalconfig"
 	"github.com/drud/ddev/pkg/nodeps"
 	"github.com/drud/ddev/pkg/output"
+	"github.com/drud/ddev/pkg/styles"
 	"github.com/drud/ddev/pkg/util"
 	"github.com/jedib0t/go-pretty/v6/table"
 	"time"
@@ -45,10 +46,6 @@ func List(activeOnly bool, continuous bool, continuousSleepTime int) {
 			if nodeps.ArrayContainsString(globalconfig.DdevGlobalConfig.OmitContainersGlobal, globalconfig.DdevRouterContainer) {
 				extendedRouterStatus = "disabled"
 			}
-			tWidth, _ := nodeps.GetTerminalWidthHeight()
-			t.SetAllowedRowLength(tWidth)
-			util.Debug("detected terminal width=%v", tWidth)
-			t.SortBy([]table.SortBy{{Name: "Name"}})
 			t.AppendFooter(table.Row{
 				"Router", "", routerStatus},
 			)
@@ -71,4 +68,52 @@ func List(activeOnly bool, continuous bool, continuousSleepTime int) {
 
 		time.Sleep(time.Duration(continuousSleepTime) * time.Second)
 	}
+}
+
+// CreateAppTable will create a new app table for describe and list output
+func CreateAppTable(out *bytes.Buffer) table.Writer {
+	t := table.NewWriter()
+	t.AppendHeader(table.Row{"Name", "Type", "Location", "URL", "Status"})
+	termWidth, _ := nodeps.GetTerminalWidthHeight()
+	usableWidth := termWidth - 15
+	t.SetAllowedRowLength(termWidth)
+	statusWidth := 7 // Maybe just "running"
+	nameWidth := 10
+	typeWidth := 7 // drupal7
+	locationWidth := 20
+	urlWidth := 20
+	if termWidth > 80 {
+		urlWidth = urlWidth + (termWidth-80)/2
+		locationWidth = locationWidth + (termWidth-80)/2
+	}
+	totUsedWidth := nameWidth + typeWidth + locationWidth + urlWidth + statusWidth
+
+	util.Debug("detected terminal width=%v usableWidth=%d statusWidth=%d nameWidth=%d typeWIdth=%d locationWidth=%d urlWidth=%d totUsedWidth=%d", termWidth, usableWidth, statusWidth, nameWidth, typeWidth, locationWidth, urlWidth, totUsedWidth)
+	t.SortBy([]table.SortBy{{Name: "Name"}})
+
+	t.SetColumnConfigs([]table.ColumnConfig{
+		{
+			Name:     "Name",
+			WidthMax: nameWidth,
+		},
+		{
+			Name:     "Type",
+			WidthMax: int(typeWidth),
+		},
+		{
+			Name:     "Location",
+			WidthMax: locationWidth,
+		},
+		{
+			Name:     "URL",
+			WidthMax: urlWidth,
+		},
+		{
+			Name:     "Status",
+			WidthMax: statusWidth,
+		},
+	})
+	styles.SetGlobalTableStyle(t)
+	t.SetOutputMirror(out)
+	return t
 }

--- a/pkg/ddevapp/list.go
+++ b/pkg/ddevapp/list.go
@@ -76,7 +76,6 @@ func CreateAppTable(out *bytes.Buffer) table.Writer {
 	t.AppendHeader(table.Row{"Name", "Type", "Location", "URL", "Status"})
 	termWidth, _ := nodeps.GetTerminalWidthHeight()
 	usableWidth := termWidth - 15
-	t.SetAllowedRowLength(termWidth)
 	statusWidth := 7 // Maybe just "running"
 	nameWidth := 10
 	typeWidth := 7 // drupal7
@@ -85,34 +84,39 @@ func CreateAppTable(out *bytes.Buffer) table.Writer {
 	if termWidth > 80 {
 		urlWidth = urlWidth + (termWidth-80)/2
 		locationWidth = locationWidth + (termWidth-80)/2
+		statusWidth = statusWidth + (termWidth-80)/3
 	}
 	totUsedWidth := nameWidth + typeWidth + locationWidth + urlWidth + statusWidth
 
 	util.Debug("detected terminal width=%v usableWidth=%d statusWidth=%d nameWidth=%d typeWIdth=%d locationWidth=%d urlWidth=%d totUsedWidth=%d", termWidth, usableWidth, statusWidth, nameWidth, typeWidth, locationWidth, urlWidth, totUsedWidth)
 	t.SortBy([]table.SortBy{{Name: "Name"}})
 
-	t.SetColumnConfigs([]table.ColumnConfig{
-		{
-			Name:     "Name",
-			WidthMax: nameWidth,
-		},
-		{
-			Name:     "Type",
-			WidthMax: int(typeWidth),
-		},
-		{
-			Name:     "Location",
-			WidthMax: locationWidth,
-		},
-		{
-			Name:     "URL",
-			WidthMax: urlWidth,
-		},
-		{
-			Name:     "Status",
-			WidthMax: statusWidth,
-		},
-	})
+	if !globalconfig.DdevGlobalConfig.SimpleFormatting {
+		t.SetAllowedRowLength(termWidth)
+
+		t.SetColumnConfigs([]table.ColumnConfig{
+			{
+				Name:     "Name",
+				WidthMax: nameWidth,
+			},
+			{
+				Name:     "Type",
+				WidthMax: int(typeWidth),
+			},
+			{
+				Name:     "Location",
+				WidthMax: locationWidth,
+			},
+			{
+				Name:     "URL",
+				WidthMax: urlWidth,
+			},
+			{
+				Name:     "Status",
+				WidthMax: statusWidth,
+			},
+		})
+	}
 	styles.SetGlobalTableStyle(t)
 	t.SetOutputMirror(out)
 	return t

--- a/pkg/ddevapp/list_test.go
+++ b/pkg/ddevapp/list_test.go
@@ -3,9 +3,11 @@ package ddevapp_test
 import (
 	"bytes"
 	"github.com/drud/ddev/pkg/ddevapp"
+	"github.com/drud/ddev/pkg/globalconfig"
 	"github.com/drud/ddev/pkg/nodeps"
 	"github.com/drud/ddev/pkg/testcommon"
 	assert2 "github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 	"os"
 	"regexp"
 	"runtime"
@@ -37,16 +39,27 @@ func TestListWithoutDir(t *testing.T) {
 	assert.NoError(err)
 
 	app, err := ddevapp.NewApp(testDir, true)
+	app.Name = t.Name()
+	_ = globalconfig.RemoveProjectInfo(app.Name)
+
 	assert.NoError(err)
+	_ = app.Stop(true, false)
+	err = os.Chdir(testDir)
+	require.NoError(t, err)
 
 	t.Cleanup(func() {
 		err = os.Chdir(origDir)
 		assert.NoError(err)
+		_ = globalconfig.RemoveProjectInfo(app.Name)
 		err = app.Stop(true, false)
 		assert.NoError(err)
 		err = os.RemoveAll(testDir)
 		assert.NoError(err)
+		globalconfig.DdevGlobalConfig.SimpleFormatting = false
+		_ = globalconfig.WriteGlobalConfig(globalconfig.DdevGlobalConfig)
 	})
+	globalconfig.DdevGlobalConfig.SimpleFormatting = true
+	_ = globalconfig.WriteGlobalConfig(globalconfig.DdevGlobalConfig)
 	app.Name = t.Name()
 	app.Type = nodeps.AppTypeDrupal7
 	err = app.WriteConfig()

--- a/pkg/ddevapp/list_test.go
+++ b/pkg/ddevapp/list_test.go
@@ -1,0 +1,95 @@
+package ddevapp_test
+
+import (
+	"bytes"
+	"github.com/drud/ddev/pkg/ddevapp"
+	"github.com/drud/ddev/pkg/nodeps"
+	"github.com/drud/ddev/pkg/testcommon"
+	assert2 "github.com/stretchr/testify/assert"
+	"os"
+	"regexp"
+	"runtime"
+	"testing"
+)
+
+// TestListWithoutDir prevents regression where ddev list panics if one of the
+// sites found is missing a directory
+func TestListWithoutDir(t *testing.T) {
+	// Can't run with mutagen because we actually delete the alpha
+	if runtime.GOOS == "windows" || nodeps.MutagenEnabledDefault == true {
+		t.Skip("Skipping because unreliable on Windows and can't be used with mutagen")
+	}
+	// Set up tests and give ourselves a working directory.
+	assert := assert2.New(t)
+	testcommon.ClearDockerEnv()
+	origDir, _ := os.Getwd()
+
+	// startCount is the count of apps at the start of this adventure
+	apps := ddevapp.GetActiveProjects()
+	startCount := len(apps)
+
+	testDir := testcommon.CreateTmpDir(t.Name())
+	defer testcommon.CleanupDir(testDir)
+
+	err := os.MkdirAll(testDir+"/sites/default", 0777)
+	assert.NoError(err)
+	err = os.Chdir(testDir)
+	assert.NoError(err)
+
+	app, err := ddevapp.NewApp(testDir, true)
+	assert.NoError(err)
+
+	t.Cleanup(func() {
+		err = os.Chdir(origDir)
+		assert.NoError(err)
+		err = app.Stop(true, false)
+		assert.NoError(err)
+		err = os.RemoveAll(testDir)
+		assert.NoError(err)
+	})
+	app.Name = t.Name()
+	app.Type = nodeps.AppTypeDrupal7
+	err = app.WriteConfig()
+	assert.NoError(err)
+
+	// Do a start on the configured site.
+	app, err = ddevapp.GetActiveApp("")
+	assert.NoError(err)
+	err = app.Start()
+	assert.NoError(err)
+
+	err = os.Chdir(origDir)
+	assert.NoError(err)
+
+	err = os.RemoveAll(testDir)
+	assert.NoError(err)
+
+	apps = ddevapp.GetActiveProjects()
+
+	assert.EqualValues(len(apps), startCount+1)
+
+	// Make a whole table and make sure our app directory missing shows up.
+	// This could be done otherwise, but we'd have to go find the site in the
+	// array first.
+	out := bytes.Buffer{}
+	table := ddevapp.CreateAppTable(&out)
+	for _, site := range apps {
+		desc, err := site.Describe(false)
+		if err != nil {
+			t.Fatalf("Failed to describe site %s: %v", site.GetName(), err)
+		}
+
+		ddevapp.RenderAppRow(table, desc)
+	}
+	table.Render()
+	assert.Regexp(regexp.MustCompile("(?s)"+ddevapp.SiteDirMissing+".*"+testDir), out.String())
+
+	err = app.Stop(true, false)
+	assert.NoError(err)
+}
+
+// TestDdevList tests the ddevapp.List() functionality
+// It's only here for profiling at this point.
+func TestDdevList(t *testing.T) {
+	ddevapp.List(true, false, 1)
+}

--- a/pkg/ddevapp/mutagen.go
+++ b/pkg/ddevapp/mutagen.go
@@ -14,7 +14,6 @@ import (
 	"github.com/drud/ddev/pkg/version"
 	docker "github.com/fsouza/go-dockerclient"
 	"github.com/pkg/errors"
-	"golang.org/x/term"
 	"os"
 	osexec "os/exec"
 	"path"
@@ -155,7 +154,7 @@ func CreateMutagenSync(app *DdevApp) error {
 	// In tests or other non-interactive environments we don't need to show the
 	// mutagen sync monitor output (and it fills up the test logs)
 
-	if term.IsTerminal(int(os.Stderr.Fd())) {
+	if os.Getenv("DDEV_NONINTERACTIVE") != "true" {
 		go func() {
 			previousStatus := ""
 			curStatus := ""

--- a/pkg/ddevapp/mutagen.go
+++ b/pkg/ddevapp/mutagen.go
@@ -269,7 +269,7 @@ func parseMutagenStatusLine(fullStatus string) string {
 
 // MutagenSyncFlush performs a mutagen sync flush, waits for result, and checks for errors
 func (app *DdevApp) MutagenSyncFlush() error {
-	if app.MutagenEnabled || app.MutagenEnabledGlobal {
+	if app.IsMutagenEnabled() {
 		syncName := MutagenSyncName(app.Name)
 		if !MutagenSyncExists(app) {
 			return errors.Errorf("Mutagen sync session '%s' does not exist", syncName)
@@ -358,7 +358,7 @@ func StopMutagenDaemon() {
 // DownloadMutagenIfNeeded downloads the proper version of mutagen
 // if it's either not yet installed or has the wrong version.
 func DownloadMutagenIfNeeded(app *DdevApp) error {
-	if !app.MutagenEnabled || app.MutagenEnabledGlobal {
+	if !app.IsMutagenEnabled() {
 		return nil
 	}
 	curVersion, err := version.GetLiveMutagenVersion()
@@ -373,7 +373,7 @@ func DownloadMutagenIfNeeded(app *DdevApp) error {
 
 // MutagenReset stops (with flush), removes the docker volume, starts again (with flush)
 func MutagenReset(app *DdevApp) error {
-	if app.MutagenEnabled || app.MutagenEnabledGlobal {
+	if app.IsMutagenEnabled() {
 		err := app.Stop(false, false)
 		if err != nil {
 			return errors.Errorf("Failed to stop project %s: %v", app.Name, err)
@@ -459,5 +459,9 @@ func IsMutagenVolumeMounted(app *DdevApp) (bool, error) {
 		}
 	}
 	return false, nil
+}
 
+// IsMutagenEnabled returns true if mutagen is enabled locally or globally
+func (app *DdevApp) IsMutagenEnabled() bool {
+	return app.MutagenEnabled || app.MutagenEnabledGlobal
 }

--- a/pkg/ddevapp/mutagen.go
+++ b/pkg/ddevapp/mutagen.go
@@ -397,7 +397,10 @@ func MutagenMonitor(app *DdevApp) {
 
 	// This doesn't actually return; you have to <ctrl-c> to end it
 	c := osexec.Command(globalconfig.GetMutagenPath(), "sync", "monitor", syncName)
+	// We only need all three of these because of Windows behavior on git-bash with no pty
 	c.Stdout = os.Stdout
+	c.Stderr = os.Stderr
+	c.Stdin = os.Stdin
 	_ = c.Run()
 }
 

--- a/pkg/ddevapp/utils.go
+++ b/pkg/ddevapp/utils.go
@@ -1,11 +1,9 @@
 package ddevapp
 
 import (
-	"bytes"
 	"fmt"
 	"github.com/drud/ddev/pkg/globalconfig"
 	"github.com/drud/ddev/pkg/nodeps"
-	"github.com/drud/ddev/pkg/styles"
 	"github.com/fsouza/go-dockerclient"
 	"github.com/jedib0t/go-pretty/v6/table"
 	"path"
@@ -55,15 +53,6 @@ func GetActiveProjects() []*DdevApp {
 	}
 
 	return apps
-}
-
-// CreateAppTable will create a new app table for describe and list output
-func CreateAppTable(out *bytes.Buffer) table.Writer {
-	t := table.NewWriter()
-	t.AppendHeader(table.Row{"Name", "Type", "Location", "URL", "Status"})
-	styles.SetGlobalTableStyle(t)
-	t.SetOutputMirror(out)
-	return t
 }
 
 // RenderHomeRootedDir shortens a directory name to replace homedir with ~

--- a/pkg/nodeps/utils.go
+++ b/pkg/nodeps/utils.go
@@ -93,7 +93,7 @@ func IsInteger(s string) bool {
 }
 
 // GetTerminalWidthHeight returns width, height if on terminal
-// or 0, 0 if not
+// or 80, 0 if not. If we can't get terminal info, we'll just assume 80x24
 func GetTerminalWidthHeight() (int, int) {
 	if term.IsTerminal(int(os.Stdout.Fd())) {
 		width, height, err := term.GetSize(int(os.Stdout.Fd()))
@@ -101,5 +101,5 @@ func GetTerminalWidthHeight() (int, int) {
 			return width, height
 		}
 	}
-	return 0, 0
+	return 80, 24
 }

--- a/pkg/styles/tables.go
+++ b/pkg/styles/tables.go
@@ -3,8 +3,6 @@ package styles
 import (
 	"github.com/drud/ddev/pkg/globalconfig"
 	"github.com/jedib0t/go-pretty/v6/table"
-	"golang.org/x/term"
-	"os"
 )
 
 var DefaultTableStyle = "StyleLight"
@@ -23,7 +21,7 @@ func GetTableStyle(name string) (style table.Style) {
 
 // SimpleFormattingRequired() returns true if we should not be colorizing/styling text
 func SimpleFormattingRequired() bool {
-	if globalconfig.DdevGlobalConfig.SimpleFormatting || !term.IsTerminal(int(os.Stdout.Fd())) {
+	if globalconfig.DdevGlobalConfig.SimpleFormatting {
 		return true
 	}
 	return false

--- a/pkg/util/utils.go
+++ b/pkg/util/utils.go
@@ -38,7 +38,7 @@ func Failed(format string, a ...interface{}) {
 	}
 }
 
-// Error will print an red error message but will not exit.
+// Error will print a red error message but will not exit.
 func Error(format string, a ...interface{}) {
 	if a != nil {
 		output.UserErr.Errorf(format, a...)

